tox.ini

```diff
@@ -34,7 +34,7 @@ commands =
 [testenv:py{37,38,39,310}-distributed]
 passenv = AWS_PROFILE AWS_DEFAULT_REGION AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN
 setenv =
-       COV_FAIL_UNDER = 75.00
+       COV_FAIL_UNDER = 74.00
```

Comment:
We have a slight issue with the way we measure coverage in dist mode. We exclude distributed [directory](https://github.com/aws/aws-sdk-pandas/blob/feat/timestream-batch-load/pyproject.toml#L185) from the calculation to avoid coverage contamination in non-dist mode but it's obviously required to measure coverage in dist mode...